animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -41,30 +55,25 @@ cdef class Mesh:
     def __cinit__(self, Vector3DList vertices = None,
                         EdgeIndicesList edges = None,
                         PolygonIndicesList polygons = None,
-                        LongList materialIndices = None,
                         bint skipValidation = False):
 
         if vertices is None: vertices = Vector3DList()
         if edges is None: edges = EdgeIndicesList()
         if polygons is None: polygons = PolygonIndicesList()
-        if materialIndices is None: materialIndices = LongList()
 
         if not skipValidation:
-            checkMeshData(vertices, edges, polygons, materialIndices)
+            checkMeshData(vertices, edges, polygons)
 
         self.vertices = vertices
         self.edges = edges
         self.polygons = polygons
-        self.materialIndices = materialIndices
 
         self.derivedMeshDataCache = {}
-        self.uvMaps = OrderedDict()
-        self.vertexColorLayers = OrderedDict()
+        self.attributes = OrderedDict()
 
     def getMeshProperties(self):
```

Comment:
This function should be removed. Loop over attributes directly.

Reply:
Okay.